cwms-data-api/src/main/java/cwms/cda/data/dto/TimeSeries.java

```diff
@@ -159,7 +164,8 @@ public ZonedDateTime getEnd() {
     }
 
     // Use the array shape to optimize data transfer to client
-    @JsonFormat(shape=JsonFormat.Shape.ARRAY)
+    @JsonFormat(shape = JsonFormat.Shape.ARRAY)
+    @JsonDeserialize(contentUsing = TimeSeriesRecordDeserializer.class)
```

Comment:
This method is overwritten for XML using a Mixin, did you verify that behavior works as intended still?

Reply:
I added a Mixin test that verifies that the XML tags for the value records have the appropriate labels. There are also a couple serialization/deserialization tests that verify that this works as intended.